### ISSUE TITLE
TalkBack, screen reader for Android devices cannot read/select the FluentSelect entries

### ISSUE THREAD

**Issue by true-perfect-code** [microsoft/fast-blazor#3993]
# 🐛 Bug Report

TalkBack, screen reader for Android devices cannot read/select the FluentSelect entries.

It Works on Web (Blazor ServeR), Windows and iOS (Maui Blazor).

See also https://github.com/microsoft/fluentui/issues/20785

## 💻 Repro or Code Sample

```
<FluentSelect Width="150px" Style="margin-left: 15px; margin-top: 7px;" @bind-Value="@FormUserSettings.Language" TOption="string">
    <FluentOption Value="EN">English</FluentOption>
    <FluentOption Value="DE">Deutsch</FluentOption>
</FluentSelect>
```

## 💁 Possible Solution

Replace with HTML list

**Comment by vnbaaij** [comment]
> TalkBack, screen reader for Android devices cannot read/select the FluentSelect entries.
> 
> It Works on Web (Blazor ServeR), Windows and iOS (Maui Blazor).

I don't understand. What works on Web, etc?

The issue you are linking to is for the Fluent UI **React** components and those a completely separate from the web components we use (but the code for both lives in the same repo).
As you probably already know, we use v2 web components which are no longer being maintained. As the bug lies there (not in our components, we 'just' wrap the web components in this case), we can't fix this til our next major release (v5) which will use the web components v3. 
I suggest you try our v5 preview packages to see if your issue is resolved. If not, please create an issue in https://github.com/microsoft/fluentui/ with the label 'web components v3'

**Comment by true-perfect-code** [comment]
@vnbaaij 

Thank you very much for the answer and the suggestion.

We are currently in the release phase and I can only report on the issues we discover during our User Acceptance Tests (UATs).

Once the current stress is over, I would like to look at FluentUI v5 version especially on accessibility as this is a big issue in the EU.

When is a v5 release planned?

pc
